### PR TITLE
@alloy => Add a web-view information overlay for admins

### PR DESCRIPTION
**Pull request by orta**
screen shot 2015-07-21 at 11 26 00

This can wait for the next release NP.

### CODE REVIEW

**Comment by alloy** [comment]
> build versions, phone OS etc

I think it’s a bit noisy and redundant, but I might not realise the use case for this well.

**Comment by orta** [comment]
Most of the times this would be used is for people taking screenshots IMO, though we can use it took obv

**Comment by ashfurrow** [comment]
An alternative would be to detect when a screenshot has been made and automatically generate an image with these details and save it to the camera roll.

**Comment by orta** [comment]
fancy

File: Artsy/View_Controllers/Web_Browsing/ARExternalWebBrowserViewController.m [hunk 1]
```diff
@@ -37,6 +37,52 @@ - (void)viewDidLoad
     [self setupConstraints];
     self.scrollView.delegate = self;
     self.scrollView.decelerationRate = UIScrollViewDecelerationRateNormal;
+
+#ifdef STORE
+#else
```
Comment (alloy) [inline Artsy/View_Controllers/Web_Browsing/ARExternalWebBrowserViewController.m]: `#ifndef STORE` ?

File: Artsy/View_Controllers/Web_Browsing/ARExternalWebBrowserViewController.m [hunk 2]
```diff
@@ -37,6 +37,52 @@ - (void)viewDidLoad
...
+    UILongPressGestureRecognizer *adminGesture;
+    adminGesture = [[UILongPressGestureRecognizer alloc] initWithTarget:self action:@selector(showAdminDetails:)];
+    [self.view addGestureRecognizer:adminGesture];
+#endif
```
Comment (alloy) [inline Artsy/View_Controllers/Web_Browsing/ARExternalWebBrowserViewController.m]: The method below can also be included in this CPP block.

File: Artsy/View_Controllers/Web_Browsing/ARExternalWebBrowserViewController.m (hunk 1 above)
Comment (orta) [inline Artsy/View_Controllers/Web_Browsing/ARExternalWebBrowserViewController.m]: weirdly, this didn't end up in the syntax highlighter, so I guessed it didn't exist :D

File: docs/BETA_CHANGELOG.md [hunk 3]
```diff
@@ -1 +1,56 @@
 ## Next
+
```
Comment (orta) [inline docs/BETA_CHANGELOG.md]: hah

File: Artsy/View_Controllers/Web_Browsing/ARExternalWebBrowserViewController.m (hunk 1 above)
Comment (alloy) [inline Artsy/View_Controllers/Web_Browsing/ARExternalWebBrowserViewController.m]: Ah, you know me, no syntax nothing editor :D

File: docs/BETA_CHANGELOG.md (hunk 3 above)
Comment (alloy) [inline docs/BETA_CHANGELOG.md]: Yah :( fixed

**Comment by alloy** [comment]
Known failures, merging!

**Pull request merged**